addon/components/country-chooser.js

```diff
@@ -4,14 +4,27 @@ import CountryCodes from '@upfluence/ember-upf-utils/resources/country-codes';
 export default class CountryChooserComponent extends Component {
   codes = CountryCodes
 
+  constructor(owner, args) {
+    super(owner, args);
+
+    if (!args.onCountrySelection) {
+      throw new Error(
+        '[component][country-chooser] Please provide a onCountrySelection action'
+      );
+    }
+  }
+
   // Named Arguments Defaults.
   // -------------------------
   get label() { return this.args.label || 'Country'; }
   get placeholder() { return this.args.placeholder || '-'; }
-  get required() { return this.args.required || false; }
```

Comment:
This was never used (in both this version or the previous Ember classic component one)